src/main.rs

```diff
@@ -1,30 +1,66 @@
 use std::{fs::File, panic, time::Duration};
 
+use clap::{Arg, ArgAction, Command};
 use crossterm::event::{Event as TuiEvent, EventStream};
 use dpp::{identity::accessors::IdentityGettersV0, version::PlatformVersion};
 use futures::{future::OptionFuture, select, FutureExt, StreamExt};
 use rs_platform_explorer::{
-    backend::{insight::InsightAPIClient, Backend},
+    backend::{self, identities::IdentityTask::{self}, insight::InsightAPIClient, wallet::WalletTask, Backend, Task},
     config::Config,
     ui::{IdentityBalance, Ui, UiFeedback},
     Event,
 };
 use rs_sdk::{RequestSettings, SdkBuilder};
-use tracing_subscriber::EnvFilter;
 
 #[tokio::main]
 async fn main() {
+    // Parse CLI command if any
+    let matches = Command::new("rs-platform-explorer")
```

Comment:
Consider following the code style we already have in other parts of the code. 

We usually create a struct and add annotations to define arguments. See drive-abci [main](https://github.com/dashpay/platform/blob/v1.0-dev/packages/rs-drive-abci/src/main.rs#L62) and load-tests main for approach.

Reply:
Done. Thanks for pointing it out.